Examples/Tests/particle_fields_diags/inputs

```diff
@@ -76,3 +76,14 @@ diag1.particle_fields_species = electrons protons photons
 diag1.particle_fields.z(x,y,z,ux,uy,uz) = z
 diag1.particle_fields.uz(x,y,z,ux,uy,uz) = uz
 diag1.particle_fields.zuz(x,y,z,ux,uy,uz) = z * uz
+
+openpmd.format = openpmd
```

Comment:
If we find ADIOS2, we will default to faster ADIOS `.bp` files.
Since the script required HDF5, we need to write this here to make it stable:
```suggestion
openpmd.format = openpmd
openpmd.openpmd_backend = h5
```